magnolia/src/test/scala/zio/config/magnolia/AutomaticConfigDescriptorSpec.scala

```diff
@@ -120,4 +120,10 @@ object AutomaticConfigTestUtils {
       case (None, Some(v2))     => partialMyConfig + (("anotherDefault", v2.toString))
       case (None, None)         => partialMyConfig
     }
+
+  def genAlpha: Gen[Random, String] =
+    for {
+      n <- Gen.int(1, 10)
+      s <- Gen.listOfN(n)(Gen.char(65, 122))
```

Comment:
There may be alphaNumeric Gen already

Reply:
Yea, Unfortunately it emitted 03 and gave and got parsed as an `Int`.